project/npda/tests/test_csv_upload.py

```diff
@@ -366,11 +423,6 @@ def test_invalid_nhs_number(test_user, single_row_valid_df):
     errors = csv_upload_sync(test_user, single_row_valid_df)
     assert "nhs_number" in errors[0]
 
-    # Catastrophic - Patient not save
```

Comment:
It's fine to save an invalid NHS number, it will be flagged in the data quality report and UI